venti/config

```diff
@@ -1,5 +1,4 @@
 SUITE=${SUITE:-buster}
 MIRROR=${MIRROR:-http://deb.debian.org/debian/}
 FLAVOUR=${FLAVOUR:-minimal}
-DUMBINIT_URL=${DUMBINIT_URL:-https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64.deb}
-PKG_INCLUDE=${PKG_INCLUDE:-apt-transport-https ca-certificates curl gcc git inetutils-ping iproute2 iptables less libc6-dev locales make openssh-client psmisc python python-dev python-pip rsync sudo unzip vim-tiny wget}
+PKG_INCLUDE=${PKG_INCLUDE:-apt-transport-https ca-certificates curl dumb-init gcc git inetutils-ping iproute2 iptables less libc6-dev locales make openssh-client psmisc python python-dev python-pip rsync sudo unzip vim-tiny wget}
```

Comment:
Not sure if the venti/config still get's used or got copied, but I'm not sure we should be shipping gcc and dev tools in the base "minimal" image.

Reply:
Venti is intended to be used as a build image -- per the [README](https://github.com/gravitational/docker-debian/blob/82c75586f13d8fe9bd596f7561290c6b76b0df3e/README.md#debian-venti-).  I think it is reasonable to have these packages included.

There also don't appear to be changes to this line compared to the previous version aside from the inclusion of `dumb-init`.  I think we should save discussion of what should be in the images for outside this PR.

Reply:
I am pretty sure gcc is needed, for example, to build golang sqlite binding for planet/gravity.
And as Walt said venti is considered as build image.